homeassistant/components/ads/sensor.py

```diff
@@ -45,28 +60,50 @@ def setup_platform(
     discovery_info: DiscoveryInfoType | None = None,
 ) -> None:
     """Set up an ADS sensor device."""
-    ads_hub = hass.data.get(ads.DATA_ADS)
-
+    ads_hub: ads.AdsHub = hass.data[ads.DATA_ADS]
```

Comment:
It would be good for maintenance if this change could be applied to all other platforms in a follow-up PR.